src/niscope/metadata/attributes.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-SCOPE API metadata version 20.0.0d13
+# This file is generated from NI-SCOPE API metadata version 20.1.0d1
```

Comment:
20.0.0f0 won't do, huh? Please confirm. If that's the case well that's too bad and alos why we don't put this version in the README.

Reply:
yeah, sadly 20.0 won't cut it.  The attributes can only made available in this version.